lib/europeana/page.rb

```diff
@@ -66,12 +68,12 @@ def media
     end
 
     def as_chapter
-      {
+      @as_chapter ||= {
         is_chapter_nav: true,
         title: @page.title,
         url: show_page_url(@page.language_code, @page.urlname),
-        label: find_thumbnail ? find_thumbnail[:label] : false,
-        image: find_thumbnail ? find_thumbnail[:image] : false
+        label: chapter_thumbnail ? chapter_thumbnail[:label] : false,
```

Comment:
`#chapter_thumbnail` always returns a hash. Either it should return `nil` when there is none, or the condition here (and on the following line) needs to be `chapter_thumbnail.present?`